core/services/keeper/validate_test.go

```diff
@@ -32,12 +32,14 @@ func TestValidatedKeeperSpec(t *testing.T) {
 		{
 			name: "valid job spec",
 			args: args{
-				tomlString: testspecs.GenerateKeeperSpec(testspecs.KeeperSpecParams{
-					Name:              "example keeper spec",
-					ContractAddress:   "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba",
-					FromAddress:       "0xa8037A20989AFcBC51798de9762b351D63ff462e",
-					ObservationSource: ExpectedObservationSource,
-				}).Toml(),
+				tomlString: `
+                        type            			= "keeper"
+                        name            			= "example keeper spec"
+                        contractAddress 			= "0x9E40733cC9df84636505f4e6Db28DCa0dC5D1bba"
+                        fromAddress     			= "0xa8037A20989AFcBC51798de9762b351D63ff462e"
+                        externalJobID   			=  "123e4567-e89b-12d3-a456-426655440002"
+						minIncomingConfirmations    = 0
```

Comment:
formatting still looks off

Reply:
curious, why did you add minIncomingConfirmations? '0' shouldn't be an allowed value. We also enforce this in env var config https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/config/config.go#L192-L194

Reply:
I think it was a confusion on my side, probably caused by the fact that the `generateKeeperSpec` function which initially was present in this test case was not setting any value for the minIncomingConfirmations field, hence leaving it to the zero value in Go, which for uint32 is 0. However, I probably misunderstood it when I had a first look at it. Updated the minIncomingConfirmations field now to be at least 1. 👍